src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -25,6 +25,8 @@ class GatlingPluginExtension implements JvmConfigurable {
 
     String logLevel = "WARN"
 
+    LogHttp logHttp = LogHttp.NONE
```

Comment:
Documentation has to be updated.

Reply:
We'll take care of the documentation in one go as we'll move it to the Gatling official doc.